### PR TITLE
add homework6

### PR DESCRIPTION
**Pull request by SvetlanaMag**

### CODE REVIEW

File: HW6/task1.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+function getEmployeeInfo(name) {
```
Comment (anatoly-karpovich) [inline HW6/task1.js:1]: please add commented tasks

File: HW6/task2.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    const set = new Set(arr)
+    return Array.from(set)
+}
+
+console.log(findDuplicate([1, 2, 8, 5, 5, 6, 6, 8]));
+
+//3 - 2 вариант
+function findDuplicate2(arr) {
+    
+    return  arr.filter((el, i, arr) => arr.indexOf(el) === i)
```
Comment (anatoly-karpovich) [inline HW6/task2.js:47]:
```suggestion
    return  arr.filter((el, i, arr) => arr.lastIndexOf(el) === i)
```

**Review by anatoly-karpovich — APPROVED** [review]
well done!

File: HW6/task2.js (hunk 2 above)
Comment (VladimirLQA) [reply HW6/task2.js:47]: Why ?

**Review by VladimirLQA — APPROVED** [review]

**Pull request merged**